x/executionlayer/test_common.go

```diff
@@ -117,32 +116,44 @@ func genesis(input testInput) []byte {
 	candidateBlock.Hash = []byte{}
 	candidateBlock.State = response.GetSuccess().PoststateHash
 
+	systemAccount := make([]byte, 32)
+	res, errStr := grpc.Query(input.elk.client, candidateBlock.State, "address", systemAccount, []string{}, genesisConfig.GetProtocolVersion())
+	if errStr != "" {
+		panic(errStr)
+	}
+
+	for _, namedKey := range res.GetAccount().GetNamedKeys() {
+		if namedKey.GetName() == types.ProxyContractName {
```

Comment:
If we are aiming to expose a payment API only through client_api_proxy, in case of failure of query `client_api_proxy` key, we need to cause panic. Because users cannot deploy a contract without this key.

Reply:
This is the same opinion as above.

Reply:
But regardless of the execution flag, because MsgUnbond, MsgBond and MsgTransfer rely on ProxyContract, they won't work if fail to query `client_api_proxy`.

Reply:
I mean [here](https://github.com/hdac-io/friday/pull/78/files#diff-5222419bc35f4f1fb5abc88dbc0379d2R55) not test_common.go. :sweat_smile:

Reply:
resolved in 491e5b2